conductorSetup.sh

```diff
@@ -0,0 +1,27 @@
+# 1) go to https://git.corp.nextdoor.com/settings/developers, and create a new OAuth app for conductor
+# replace the Client Id and Client Secret in the variable below
+OAUTH_CLIENT_ID='YOUR_OAUTH_CLIENT_ID'
+
+PINK='\033[0;35m'
+NC='\033[0m'        # No Color
+
+# echo -e "${PINK}checking install of package management tools..${NC}"
+if ! brew ls --versions yarn; then install yarn; fi; 
+
+echo -e "${PINK}bringing up new postgres docker container for conductor...${NC}"
+make postgres
+echo -e "${PINK}sleeping for 5 seconds before connecting with new postgres instance...${NC}"
+sleep 5
```

Comment:
there is probably a way to check if it is running by checking the port, or looking at the service... but ok for a dev script.

Reply:
Eric said that's how we generally do it for our setup scripts. And it solves the problem here. if the check command would be like, while not active on port, sleep 1.